src/plugins/packages.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+const fs = require("fs");
+const colors = require("colors/safe");
+const Helper = require("../helper");
+const themes = require("./themes");
+
+const customCss = [];
+
+module.exports = {
+	getCustomCss: getCustomCss,
+	loadPackages: loadPackages,
+};
+
+const packageApi = function(packageName) {
+	return {
+		addCustomCss: addCustomCss.bind(this, packageName),
+	};
+};
+
+function addCustomCss(packageName, filename) {
+	customCss.push(packageName + "/" + filename);
+}
+
+function getCustomCss() {
+	return customCss;
+}
+
+function loadPackages() {
+	fs.readdir(Helper.getPackagesPath(), (err, packages) => {
+		if (err) {
+			return;
+		}
+		packages
+			.forEach((packageName) => {
+				const packageFile = getModuleInfo(packageName);
+				if (packageFile.type === "theme") {
+					themes.addtheme(packageName, packageFile);
+				}
+
+				if (packageFile.run) {
+					packageFile.run(packageApi(packageName));
+				}
+			});
+	});
+}
+
+function getModuleInfo(packageName) {
+	let module;
+	try {
+		module = require(Helper.getPackageModulePath(packageName));
+	} catch (e) {
+		log.warn(`Specified package ${colors.yellow(packageName)} is not installed in packages directory`);
+		return;
+	}
+	if (!module.thelounge && !module.lounge) {
```

Comment:
You can remove `module.lounge`, it's not there on master.

Reply:
We would have deprecated this first because we publicised it as this. I'll add it back in here with deprecation warning and we can remove it in 3.0.

Reply:
Nah, you can just remove it: we never made a release without it, it was in `master` for like 24 hours. It's silly to add something directly deprecated :/

Reply:
Alrighty

Reply:
Done